server/config/passport-setup.js

```diff
@@ -1,13 +1,29 @@
 const passport = require('passport')
 const GoogleStrategy = require('passport-google-oauth20')
+const { User } = require('../db/db').models
+
+passport.serializeUser((user, done) => {
+    done(null, user.id)
+})
+
+passport.deserializeUser(async (id, done) => {
+    const user = await User.findByPk(id)
+    done(null, user.dataValues)
+})
 
 passport.use(
     new GoogleStrategy({
         clientID: process.env.GOOGLE_CLIENT_ID,
         clientSecret: process.env.GOOGLE_CLIENT_SECRET,
-        callbackURL: 'api/auth/google/redirect'
-    }, (accessToken, refreshToken, profile, done) => {
-        console.log('passport callback!')
-        console.log({accessToken, refreshToken, profile, done})
+        callbackURL: '/api/auth/google/redirect'
+    }, async (accessToken, refreshToken, profile, done) => {
+        let googleId = profile.id
+        let user = await User.findOne({ where: { googleId }})
```

Comment:
Make sure to revisit this when we add facebook auth (in case we create a new user based on just google id or fb id)